### PR TITLE
WIP : PHP8

### PR DESCRIPTION
**Pull request by iruzevic**
⚠️ PHP8 preparation branch

# Description
- updating composer packages
- removing unnecessary standards and ignoring lines.
- updating composer command names
- updating npx command readme
- adding better copy for checkAttr helper if key is missing

## Note

To install and test the latest version, use this command:
```sh
npx create-wp-project@latest --eightshiftLibsBranch="develop-php8" --eightshiftFrontendLibsBranch="develop-php8" --eightshiftBoilerplateBranch='develop-php8'
```

## Connected PRs
https://github.com/infinum/eightshift-boilerplate/pull/273
https://github.com/infinum/eightshift-frontend-libs/pull/761
https://github.com/infinum/eightshift-libs/pull/381

### CODE REVIEW

**Review by vedranpetrac — APPROVED** [review]

**Review by iobrado — DISMISSED** [review]

**Review by iobrado — APPROVED** [review]

**Review by kancijan — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -24,15 +24,15 @@
 		"source": "https://github.com/infinum/eightshift-frontend-libs"
 	},
 	"require": {
-		"php": "^7.4 || <8.0",
+		"php": "^7.4 || >=8.0",
```
Comment (goranalkovic-infinum) [inline composer.json:27]:
```suggestion
		"php": ">=7.4",
```
we can always change it to `>=7.4 <= 8.2` for example if 8.3 adds breaking changes

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (iruzevic) [reply composer.json:27]: @dingo-d what do you think?

**Review by iobrado — APPROVED** [review]

**Review by dadadavorin — DISMISSED** [review]

File: scripts/editor/attributes.js [hunk 2]
```diff
@@ -146,12 +146,18 @@ export const checkAttr = (key, attributes, manifest, undefinedAllowed = false) =
 	// Check current component attributes.
 	const manifestKey = manifest.attributes[key];
 
+	let tipOutput = '';
+
+	if ('components' in manifest) {
+		tipOutput = ' If you are using additional components, check if you used the correct block/component prefix in your attribute name.';
```
Comment (piqusy) [inline scripts/editor/attributes.js:152]: Is the blank space at the beginning of the string intentional?

**Review by piqusy — APPROVED** [review]

**Review by vedranpetrac — DISMISSED** [review]

File: scripts/editor/attributes.js (hunk 2 above)
Comment (goranalkovic-infinum) [reply scripts/editor/attributes.js:152]: @piqusy probably, as couple of lines below you can find
```
... nent manifest.${tipOutput} Please check ...
```

File: composer.json (hunk 1 above)
Comment (dingo-d) [reply composer.json:27]: Yeah, that should work 👍🏼

**Review by dingo-d — APPROVED** [review]

**Review by kancijan — APPROVED** [review]

**Review by goranalkovic-infinum — DISMISSED** [review]

**Pull request merged**